docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -1,5 +1,6 @@
 package com.nirima.jenkins.plugins.docker;
 
+import com.github.dockerjava.api.model.*;
```

Comment:
Nobody cares about such info. If you really interesting about class hierarchy and their usage, then open in IDE.